projects/extension/src/background/AppMediator.ts

```diff
@@ -98,6 +107,13 @@ export class AppMediator extends (EventEmitter as { new(): StateEmitter }) {
     return this.#appName;
   }
 
+  /** healthStatus returns the latest health status
+   * of app as set from the callback
+   */
+  get healthStatus(): smoldot.SmoldotHealth {
```

Comment:
Why not add `SmoldotHealth` to the other import then you can avoid the repetitious `smoldot.SmoldotHealth`? It's not a blocker but it would be neater